client/src/pages/Profile/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import Layout from '../../Layout/Layout';
+import { sampleData } from './sampleData';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import ProfileDetailCard from './ProfileDetailCard/ProfileDetailCard';
+import BookingCard from './BookingCard/BookingCard';
+import { Sitter } from './sampleData';
+import useStyles from './useStyles';
+
+const ProfileDetail = (): JSX.Element => {
+  const classes = useStyles();
+  const params: { sitterId: string } = useParams();
+  const { sitterId } = params;
+
+  const [sitter, setSitter] = useState<Sitter | null | undefined>(null);
+
+  const getSitterInfo = (sitterId: string) => {
+    return sampleData.find((sitter: Sitter) => sitter.sitterId === sitterId);
+  };
+
+  useEffect(() => {
+    const sitter = getSitterInfo(sitterId);
+    setSitter(sitter);
+  }, [sitterId]);
+
+  const theme = useTheme();
+
+  const matches = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
If possible, shift this to useStyles file

Reply:
Applied